webrtc.html

```diff
@@ -5059,9 +5070,6 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
               as <code>recvonly</code> or <code>inactive</code>,
               as defined in <span data-jsep="subsequent-offers">
               [[!JSEP]]</span>. <!-- onended --></p>
-              <p>When the other peer stops sending a track in this manner, an
-              <code title="event-MediaStreamTrack-ended">ended</code> event is
-              fired at the <code><a>MediaStreamTrack</a></code> object.</p>
```

Comment:
Should this be replaced with "the track is removed from any remote streams it was added to"?